CHANGELOG.md

```diff
@@ -1,20 +1,121 @@
 ## HEAD
 
+## 0.15.0-alpha.1
+###### _Feb 23, 2016_
+
+This release includes huge improvements to the implementation
+of components and utility modules. The most important improvement
+is the removal of mixins from the library, thanks to the
+[great efforts](https://github.com/callemall/material-ui/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Anewoga+style-propable)
+of @newoga :+1:
+
+There are also improvements to the unit testing infrastructure. We own this
+great improvement to @nathanmarks, thanks a lot :+1:.
+
+Please note that `raw-themes` are deprecated with no warning! they will be removed
```

Comment:
That won't do. We also have them on the muiThemeObject. we can't warn users about that one :(